lib/lbt/analyzer/XMLTemplateAnalyzer.js

```diff
@@ -402,7 +402,7 @@ class XMLTemplateAnalyzer {
 					}
 				}
 			}
-		} catch {
+		} catch (_error) {
```

Comment:
I'm confused now. Why do we prefer this solution over the previous one (not defining any variable if it is not used)? My understanding was that the rule makes sense in general, and sometimes we need to define a parameter but do not use it. In this case it is not needed. I also tested omitting the error variable in TypeScript (ui5-linter) and could not find any problem with such a solution.

Reply:
I also endorse omitting variables in a way like this and not setting an extra rule for ignore patterns. If Typescript or any other thing does not report any errors from that we shouldn't add this here.

Reply:
This was a misunderstanding. Omitting the error variable also works fine in ui5-linter (TS). So then let's not use those new rules for now and go back to the solution from https://github.com/SAP/ui5-builder/pull/1076/commits/a5cfb8ece4e03553271a41700c59c29d82952c76. We can think about enabling the rules in a follow-up activity, but I think we shouldn't do that now when we mainly focus on doing the v9 upgrade.